CHANGELOG.md

```diff
@@ -2,12 +2,14 @@
 
 ## UNRELEASED
 
+This is a significant update that transforms this module into an ECMAScript module and changes the name of the exported function.
```

Comment:
For that reason we have updated the major version number.

Reply:
Added.